nucypher/blockchain/eth/clients.py

```diff
@@ -54,7 +54,8 @@ class Web3ClientUnexpectedVersionString(Web3ClientError):
     100: "xDai",
     137: "Polygon/Mainnet",
     11155111: "Sepolia",
-    80001: "Polygon/Mumbai"
+    80001: "Polygon/Mumbai",
```

Comment:
Is Mumbai used/supported at all?

Reply:
This is just a dictionary of all known public chains, not chains that are actually supported. Similar to `POA_CHAINS` defined lower down.

It can probably be pruned later on - "Ropsten", "Rinkeby" etc. are still in there too 😅 .

Reply:
Filed https://github.com/nucypher/nucypher/issues/3474